flang/lib/Lower/OpenMP.cpp

```diff
@@ -8,15 +8,80 @@
 
 #include "flang/Lower/OpenMP.h"
 #include "flang/Lower/Bridge.h"
+#include "flang/Lower/FIRBuilder.h"
 #include "flang/Lower/PFTBuilder.h"
 #include "flang/Parser/parse-tree.h"
+#include "mlir/Dialect/OpenMP/OpenMPDialect.h"
 
 #define TODO() llvm_unreachable("not yet implemented")
 
+void Fortran::lower::genOMP(
+    Fortran::lower::AbstractConverter &absConv,
+    Fortran::lower::pft::Evaluation &eval,
+    const Fortran::parser::OpenMPSimpleStandaloneConstruct
+        &simpleStandaloneConstruct) {
+  const auto &directive =
+      std::get<Fortran::parser::OmpSimpleStandaloneDirective>(
+          simpleStandaloneConstruct.t);
+  switch (directive.v) {
+  default:
+    TODO();
```

Comment:
The leading `default:` appears odd stylistically.

I'd suggest actually writing all the `case`s for the enum here rather than using a `default`. The benefits to not using a `default` being that it's clearer how much work needs to be done. Also, if there are changes to the enum member values, then the compiler can warn that this code does not handle them and/or needs to be updated in some manner.

Reply:
> The leading default: appears odd stylistically.

That was LLVM style. Anyways your point is more reasonable:
> The benefits to not using a default being that it's clearer how much work needs to be done.

I'll flip to your approach, and revise soon. Thanks!

Reply:
Great. Thanks. I was referring to this https://llvm.org/docs/CodingStandards.html#don-t-use-default-labels-in-fully-covered-switches-over-enumerations

You are right that default appears at the top often. It looks strange, but it's not wrong.  A brief sampling of MLIR's code shows default appears at the bottom of switches often, but at the top as well.